ParkingApplication.api/Repositories/AccountRepository.cs

```diff
@@ -1,21 +1,43 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
+using System.Data.SqlClient;
 using ParkingApplication.Api.Interfaces;
 using ParkingApplication.Api.Models;
+using Microsoft.Extensions.Options;
+using ParkingApplication.Api.Configuration;
 
-namespace Repositories;
+namespace Repositories
+{
 public class AccountRepository : IQueryRepository<AccountModel>
 {
-private readonly List<AccountModel> _accounts = new List<AccountModel>
+private readonly string _connectionString;
+
+public AccountRepository(IOptions<ConnectionStrings> connectionStrings)
 {
-    new AccountModel(1, 500, 1),
-    new AccountModel(2, 200, 2)
-};
+    _connectionString = connectionStrings.Value.DefaultConnection;
+}
 
 public List<AccountModel> GetAll()
 {
-    return _accounts;
+var accounts = new List<AccountModel>();
+
+using (var connection = new SqlConnection(_connectionString))
```

Comment:
tu acum faci cate  o conexiune in fiecare repository in loc sa se faca in program si sa o pasezi in jos cum am povestit

Reply:
ok!